src/main/resources/application.yml

```diff
@@ -50,6 +56,13 @@ application:
       programme-membership: ${PROGRAMME_MEMBERSHIP_QUEUE_URL:}
       record: ${RECORD_QUEUE_URL:}
       request: ${REQUEST_QUEUE_URL:}
+      publish:
```

Comment:
Maybe I'm having a momentary lapse of reason (it is Friday, after all!)... I can't think why we need this if we're using SNS?

Reply:
Is the idea to get the DMS changes from TIS pushed to separate queues, instead of all to 'record'?

Reply:
Wait wait, I think I get it - it's to capture the changes that need to cascade, right? (Ok, last guess!)

Reply:
Ah, sorry. Initially I was thinking it would be straight to SQS until I saw the existing implementation, I'll clean up that bit of config.